Makefile

```diff
@@ -60,7 +60,13 @@ lint:
 	pre-commit run --all-files --show-diff-on-failure
 
 test:
-	pytest --cov=vorta
+	nox -- --cov=vorta
+
+test-unit:
+	nox -- tests/unit
+
+test-integration:
+	nox -- tests/integration
```

Comment:
Should we pass coverage by default for these? What do you think?

Reply:
I didn't add them since integration might not reach full coverage but now that I think again, it might be useful to know the stats.